controllers/register.go

```diff
@@ -10,6 +10,13 @@ import (
 	"github.com/gutakk/go-google-scraper/models"
 )
 
+const (
+	registerTitle = "Register"
+	registerView  = "register"
+
+	registerSuccessfully = "Register successfully"
```

Comment:
A better way would be to use localization but I realize that it's not supported by Gin and it would require a custom [middleware](https://github.com/gin-gonic/gin/issues/1382#issuecomment-670822818). It's out of the scope of this task but it's something you could check later.

Reply:
Should I18n be the part of this PR? if no should I create another issue for this?

Reply:
I don't think it's in the scope of this project. You could create an issue to check that out later. We should always use this certification to learn as much as possible 😁

Reply:
Small English adjustment needed though:

```suggestion
	registerSuccessFlash = "Registration completed successfully"
```

Reply:
Fixed in